MAIN_tutorial_intro_to_nilearn.ipynb

```diff
@@ -510,7 +510,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.5"
+   "version": "3.6.7"
```

Comment:
Is there a reason for this increment, or was it just your local environment ?

Reply:
I think docker upgraded a new version for python (from december https://www.python.org/downloads/release/python-367/)
This is dependent on the inheritance from `jupyter/datascience-notebook`